app/src/main/java/org/oppia/android/app/topic/lessons/StorySummaryViewModel.kt

```diff
@@ -55,6 +66,11 @@ class StorySummaryViewModel(
     )
   }
 
+  private fun computeContentDescriptionText(): String {
+    return "$storyProgressPercentageText stories " +
+      "completed and $chaptersInProgress chapters in progress."
```

Comment:
This needs to go in strings.xml so that it can be translated. There are other uses of variable interpolation in strings that can be referenced in the codebase.

Note that I don't think this would replace topic_story_progress_percentage since it seems like we also need to use a plural string in this case since 'chapter' can be plural.